README.markdown

```diff
@@ -568,6 +570,12 @@ and then load the library in Lua:
     local server = require "resty.websocket.server"
 ```
 
+if you are using Debian package:
+
+```lua
+    local server = require "nginx.websocket.server"
```

Comment:
Sorry, I hate the way the package changes the Lua module name, which is breaking API compatibility.